api/src/db/migrations/v107_refactor_interstate_transport_essential_question.test.ts

```diff
@@ -0,0 +1,134 @@
+        },
+        version: 107,
+      });
+    });
+
+    it("sets the interStateLogisticsApplicable onboarding question to false for existing users that answered no", () => {
+      const v106 = generateV106UserData({
+        profileData: generateV106ProfileData({ industryId: "logistics", interstateTransport: false }),
+      });
+      const v107 = migrate_v106_to_v107(v106);
+      expect(v107).toEqual({
+        ...v106,
+        profileData: {
+          ...v106.profileData,
+          isInterstateLogisticsApplicable: false,
+        },
+        version: 107,
+      });
+    });
+
+    it("sets the interStateLogisticsApplicable onboarding question to undefined for non-logistics existing users", () => {
+      const v106 = generateV106UserData({
+        profileData: generateV106ProfileData({ industryId: "generic", interstateTransport: undefined }),
+      });
+      const v107 = migrate_v106_to_v107(v106);
+      expect(v107).toEqual({
+        ...v106,
+        profileData: {
+          ...v106.profileData,
+          isInterstateLogisticsApplicable: undefined,
+        },
+        version: 107,
+      });
+    });
+
+    it("sets the interstateLogisticsApplicable question to undefined for any new user", () => {
+      const v106 = generateV106UserData({
+        profileData: generateV106ProfileData({}),
+      });
+      const v107 = migrate_v106_to_v107(v106);
+      expect(v107).toEqual({
+        ...v106,
+        profileData: {
+          ...v106.profileData,
+          isInterstateLogisticsApplicable: undefined,
+        },
+        version: 107,
+      });
+    });
+  });
+
+  describe("interstateMoving", () => {
+    it("sets interstateMovingApplicable onboarding question to true for existing users that answered yes", () => {
+      const v106 = generateV106UserData({
+        profileData: generateV106ProfileData({ industryId: "moving-company", interstateTransport: true }),
+      });
+      const v107 = migrate_v106_to_v107(v106);
+      expect(v107).toEqual({
+        ...v106,
+        profileData: {
+          ...v106.profileData,
+          isInterstateMovingApplicable: true,
+        },
+        version: 107,
+      });
+    });
+
+    it("sets interstateMovingApplicable onboarding question to false for existing users that answered no", () => {
+      const v106 = generateV106UserData({
+        profileData: generateV106ProfileData({ industryId: "moving-company", interstateTransport: false }),
+      });
+      const v107 = migrate_v106_to_v107(v106);
+      expect(v107).toEqual({
+        ...v106,
+        profileData: {
+          ...v106.profileData,
+          isInterstateMovingApplicable: false,
+        },
+        version: 107,
+      });
+    });
+
+    it("sets interstateMovingApplicable onboarding question to undefined for any non-moving company existing users", () => {
+      const v106 = generateV106UserData({
+        profileData: generateV106ProfileData({
+          industryId: "moving-company",
```

Comment:
The test title says "non-moving-company users" but the industry here **is** moving-company?

Reply:
I wanted to account for existing users who are not in the moving-company industry.

Reply:
Gotcha, shouldn't this line that sets industryId set it to something else? Right now it is testing a user whose industry is moving-company